tests/test_movies.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from moviecenter import movies
+from moviecenter.movies import Movie
+
+
+@pytest.fixture
+def test_movie_1() -> Movie:
+    return Movie(name='LoTR', director='Steve', budget=100000000)
+
+
+@pytest.fixture
+def test_movie_2() -> Movie:
+    return Movie(name='Batman v Superman', director='Zack', budget=100000000)
+
+
+@pytest.fixture
+def test_movies(test_movie_1, test_movie_2) -> list[Movie]:
+    return [test_movie_1, test_movie_2]
+
+
+@pytest.fixture(autouse=True)
+def populate_db(test_movies):
+    movies.movies = test_movies
```

Comment:
That's just nasty - the best practise is to have your DB accessor methods accept a connection as the first argument or something